app/controllers/posts_controller.rb

```diff
@@ -1,15 +1,15 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = Post.where(author: @user)
+    @posts = Post.includes(:author).where(author: @user)
     respond_to do |format|
       format.html { render 'posts/index' }
       format.json { render json: { user: @user, posts: @posts } }
     end
   end
 
   def show
-    @post = Post.find(params[:id])
+    @post = Post.includes(comments: [:user]).find(params[:id])
```

Comment:
- [ ] Good job by fixing the N+1 problem when fetching all posts and their comments without using the `bullet gem. However as required for the project, kindly ensure that your PR description includes the screenshots of your app's logs **before** and **after** the fix